hardware/asic/skywater/openram_config.py

```diff
@@ -4,9 +4,10 @@
 num_rw_ports = 1
 num_r_ports = 0
 num_w_ports = 0
-
+#write size for adding byte enable for sram
+write_size =8
 # Number of words in the memory
-num_words = 64
+num_words = 32
```

Comment:
why 32 words ? This is possibly not enough for hello world

Reply:
this is for test only I will change it as the flow progresses